### ISSUE TITLE
rawnetworkfs defaultFS should point to real path?

### ISSUE THREAD

**Issue by chu11** [LLNL/magpie#243]
Just noticed

```
<property>
  <name>mapreduce.task.tmp.dir</name>
  <value>${fs.defaultFS}/mapred/temp</value>
  </description>
</property>
```

but fs.defaultFS is "file:///" for rawnetworkfs.  Is this an issue?  TBD

**Comment by chu11** [comment]
should I in lib/magpie-lib-setup?

```
-        magpie_fsdefault="file:///"
+        magpie_fsdefault="file://${magpie_hadooptmpdir}"
```

**Comment by chu11** [comment]
fixed in 1.82
